esi_ui/static/dashboard/esi/lessee/offers/esi-offers-table.controller.js

```diff
@@ -69,6 +70,15 @@
         });
       });
     }
+
+    function getStatusColor(availabilityStart) {
```

Comment:
Can you change this function name to something like `getOfferAvailabilityStatusColor`? Just to make it descriptive and easy to understand.